themes-default/slim/views/vue-components/app-link.mako

```diff
@@ -22,80 +23,109 @@ Vue.component('app-link', {
         placeholder: {
             type: String,
             default: 'indexer-to-name'
-        }
+        },
+        base: String
     },
     computed: {
+        config() {
+            return this.$store.state.config;
+        },
+        computedBase() {
+            // Return prop before HTML element to allow tests to mock
+            if (this.base) {
+                return this.base;
+            }
+
+            return document.getElementsByTagName('base')[0].getAttribute('href');
+        },
+        computedHref() {
+            const { indexerId, placeholder } = this;
+            if (indexerId && placeholder) {
+                return this.href.replace(placeholder, MEDUSA.config.indexers.indexerIdToName(indexerId));
+            }
+            return this.href;
+        },
+        isIRC() {
+            return this.computedHref.startsWith('irc://');
+        },
+        isAbsolute() {
+            const href = this.computedHref;
+            return /^[a-z][a-z0-9+.-]*:/.test(href);
+        },
+        isExternal() {
+            const base = this.computedBase;
+            const href = this.computedHref;
+            return !href.startsWith(base) && !href.startsWith('webcal://');
+        },
+        isHashPath() {
+            return this.computedHref.startsWith('#');
+        },
+        isAnonymised() {
```

Comment:
It can already be removed. No references remain (on `href` attributes in **`mako`** files).
I'll remove the last one which is the Donations link in another PR (I also changed it on #4519)
It didn't raise an error because it uses this method:
```mako
<app-link href="${app.DONATIONS_URL}" rel="noreferrer" onclick="window.open('${app.ANON_REDIRECT}' + this.href); return false;">
```